src/Uno.CrossTargetting.props

```diff
@@ -23,8 +23,8 @@
 	<NoWarn>$(NoWarn);0105</NoWarn>
   </PropertyGroup>
 
-	<Import Project="uno.mobile-cross-targeting-disable.props" />
```

Comment:
Maybe using a condition would be enough (using a property defined in `uno.mobile-cross-targeting-disable.props`)

Reply:
We're going to remove this once we have net7 fully included (in 17.4)